bin/evm2wasm.js

```diff
@@ -2,16 +2,71 @@
 
 const evm2wasm = require('../index.js')
 const ethUtil = require('ethereumjs-util')
+const async = require('async')
+const argv = require('minimist')(process.argv.slice(2))
+const fs = require('fs')
 
-const input = ethUtil.toBuffer(process.argv[2])
+let input, output, outputFile
 
-evm2wasm.evm2wasm(input, {
-  stackTrace: process.argv[3] === 'trace',
-  tempName: 'temp',
-  inlineOps: true,
-  wabt: true
-}).then(function (output) {
-  console.log(output.toString('binary'))
-}).catch(function (err) {
-  console.err('Failed: ' + err)
-})
+async.waterfall([
```

Comment:
How about using promises only (check promisify in wabt.js)?

Reply:
Done.